### PR TITLE
Open Recents as a gesture option

### LINKED ISSUE #3529: [FEATURE] Recents on home button tap
### Describe the feature

An option to open recents when using gestures, specifically on Home Button Tap.

### Why would it be useful to add?

People without quick switch can access recents without lag.

### Did the feature exist in Lawnchair v2? (Play Store version)

No

### Example(s)

In Nova, you can open recents using Home Button Tap and it works instantly without root acces.

Screenshot_20230705_035227_Nova7

### Additional context

I hope it is a minor feature enough to implement since Lawnchair already supports Home Button Tap. However, this feature probably needs accessibility as I observed in Nova, so user should probably be redirected to accessibility page to give accessibility permission to Lawnchair.

### LINKED ISSUE #4671: [FEATURE] ADD "OPEN RECENTS" TO GESTURES PAGE
### Describe the feature

Add an option to use "recent apps" when pressing home button

### Why would it be useful to add?

For the ones that cant use Quickswitch for gestures.

### Did the feature exist in Lawnchair v2? (Play Store version)

No

### Example(s)

Screenshot_20240819_124055_Nova Launcher

### Additional context

Thanks in advance.

### PR DESCRIPTION
**Pull request by Hyphrio**
## Description

Added a way to open Recents within the Gestures menu, which could allow Lawnchair to gain faster app switching on broken skins of Android when the user is using gestures as navigation controls.
Right now, some areas of the code look very ugly due to having to edit it without IDEA.

- Fixes #3529.
- Fixes #4671.

## Type of change

:x: General change (non-breaking change that doesn't fit the below categories like copyediting)
:x: Bug fix (non-breaking change which fixes an issue)
✅ New feature (non-breaking change which adds functionality)
:x: Breaking change (fix or feature that would cause existing functionality to not work as expected)

### CODE REVIEW

**Comment by MrSluffy** [comment]
This looks good, please fix conflict so I can test and merge it Thanks!

File: lawnchair/src/app/lawnchair/gestures/handlers/RecentsGestureHandler.kt [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+import app.lawnchair.ui.ModalBottomSheetContent
+import app.lawnchair.util.requireSystemService
+import app.lawnchair.views.ComposeBottomSheet
+import com.android.launcher3.R
+import com.android.launcher3.Utilities
+
+class RecentsGestureHandler(context: Context) : GestureHandler(context) {
+
+    override suspend fun onTrigger(launcher: LawnchairLauncher) {
+        val app = context.lawnchairApp
```
Comment (MrSluffy) [inline lawnchair/src/app/lawnchair/gestures/handlers/RecentsGestureHandler.kt]: you can use launcher here

**Review by SuperDragonXD — COMMENTED** [review]
Hello. If you're unable edit this code via IDEA (or do you mean Android Studio?), I'd suggest to use VSCode (or github.dev) with the Kotlin Language Server or similar extension, to ensure that you don't have syntax errors and similar.

**Comment by Hyphrio** [comment]
This should be compiling, I guess I broke something when I was merging branches

**Review by MrSluffy — APPROVED** [review]

**Comment by UltraBlackLinux** [comment]
Don't wanna sound like a dick here, but please amend commits instead of making a ton of small but connected commits -  or make their names more descriptive at least. This clutters up the commit history with a ton of rather confusing commits.
Sure, I haven't commited to this repository yet, but I just wanna put it out there as it makes it easier for other people to work with the codebase ;)

**Pull request merged**